internal/cli/api_test.go

```diff
@@ -0,0 +1,87 @@
+package cli
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAPICmdInputs_FromArgs(t *testing.T) {
+	const testDomain = "example.auth0.com"
+	var testCases = []struct {
+		name           string
+		givenArgs      []string
+		givenDataFlag  string
+		expectedMethod string
+		expectedURL    string
+		expectedError  string
+	}{
+		{
+			name:           "it can correctly parse input arguments",
+			givenArgs:      []string{"get", "/tenants/settings"},
```

Comment:
Given that the users will most likely not be including the leading forward slash, we should not include in the test inputs as well. Except maybe in one case, to test that it works both ways.